src/common/queries/account.ts

```diff
@@ -1,23 +1,26 @@
-import { Account } from '../../account'
-import { Address } from 'viem'
+import { Address, PublicClient } from 'viem'
 import { GRAPHQL_API_URL } from '../constants'
+import { Account } from '../../account'
 
 const query = `
-    query ($smartAccount: String) {
-      moduleQueries (where: { smartAccount: $smartAccount, isInstalled: true }) {
-        module,
+    query ($smartAccount: String, $chainId: Int) {
```

Comment:
this doesnt actually work yet though right? we're still waiting for wildcard indexing right

Reply:
@kopy-kat Yes correct